source/taggedalgebraic/taggedunion.d

```diff
@@ -202,6 +202,23 @@ align(commonAlignment!(UnionKindTypes!(UnionFieldEnum!U))) struct TaggedUnion
 		assert(false); // never reached
 	}
 
+	/// Enables using a tagged union value as an associative array key.
+	size_t toHash()
+	const {
```

Comment:
I suggest changing this line to:

```suggestion
	const nothrow @safe {
```

As otherwise the build fails because of the warning:

> source/taggedalgebraic/taggedunion.d(206,9): Warning: `toHash()` must be declared as `extern (D) size_t toHash() const nothrow @safe`, not `const nothrow @system ulong()`

Even if you want to want to let the code remain generic and support that have non-`@safe` or non-`nothrow` types, I think it would still be used to at least temporary change this line, so you can see for exactly which types attribute inference fails and why.

Reply:
Makes sense!